Formula/git-secret.rb

```diff
@@ -15,12 +17,44 @@ class GitSecret < Formula
   depends_on :gpg => :recommended
 
   def install
+    # Prefer GPG2 by default.
+    inreplace "src/_utils/_git_secret_tools.sh", ':="gpg"', ':="gpg2"'
```

Comment:
I think it'd be fine to just make them conflict so you can't have them installed side-by-side.